linux/package_linux.jl

```diff
@@ -37,22 +37,22 @@ packages = [
 artifact_hash, tarball_path, = debootstrap(arch, image; archive, packages) do rootfs, chroot_ENV
     my_chroot(args...) = root_chroot(rootfs, "bash", "-c", args...; ENV=chroot_ENV)
 
-    if arch == "x86_64"
-        host_triplet = "x86_64-linux-gnu"
+    if arch ∈ ("x86_64", "i686")
+        host_triplet = "$(arch)-linux-gnu"
         # Install GCC 9 from Elliot's repo
-        repo_release_url = "https://github.com/staticfloat/linux-gcc-toolchains/releases/download/GCC-v9.1.0-x86_64-linux-gnu"
+        repo_release_url = "https://github.com/staticfloat/linux-gcc-toolchains/releases/download/GCC-v9.1.0-$(host_triplet)"
         gcc_install_cmd = """
         cd /usr/local
-        curl -L $(repo_release_url)/GCC.v9.1.0.$(host_triplet)-target_libc+glibc-target_os+linux-target_arch+x86_64.tar.gz | tar zx
-        curl -L $(repo_release_url)/Binutils.v2.38.0.x86_64-linux-gnu-target_libc+glibc-target_os+linux-target_arch+x86_64.tar.gz | tar zx
+        curl -L $(repo_release_url)/GCC.v9.1.0.$(host_triplet)-target_libc+glibc-target_os+linux-target_arch+$(arch).tar.gz | tar zx
+        curl -L $(repo_release_url)/Binutils.v2.38.0.x86_64-linux-gnu-target_libc+glibc-target_os+linux-target_arch+$(arch).tar.gz | tar zx
```

Comment:
On line 47, need to change `x86_64` to `$(arch)`?

Reply:
```suggestion
        curl -L $(repo_release_url)/Binutils.v2.38.0.$(host_triplet)-target_libc+glibc-target_os+linux-target_arch+$(arch).tar.gz | tar zx
```